File: UriJudge/CSharp/1002/Program.cs
using System;
using System.Globalization;

namespace _1002
{
    class Program
    {
        static void Main(string[] args)
        {
            double raio = Double.Parse(Console.ReadLine(), CultureInfo.InvariantCulture);
            double n = 3.14159d;
            double area = n * Math.Pow(raio, 2);

            System.Console.WriteLine($"A={area.ToString("f4")}",CultureInfo.InvariantCulture);
        }
    }
}

File: UriJudge/C/1002/Program.c
#include <stdio.h>
#include <math.h>

int main() 
{
    double raio;
    scanf("%lf",&raio);

    double pi = 3.14159;
    raio *= raio;
    double area = round((pi * raio) * 10000) / 10000;

    printf("A=%.4lf\n", area);

    return 0; 
}
